### ISSUE TITLE
Helm: Kubernetes cluster unreachable

### ISSUE THREAD

**Issue by nicholastmosher** [infinyon/fluvio#340]
When performing a fresh install of Fluvio on a brand-new Ubuntu 20.04 VM, I saw the following error:

```
$ FLV_CMD=true fluvio cluster install
...
>> helm list --filter fluvio-sys --output json
error: Failed to parse JSON from helm output: EOF while parsing a value at line 1 column 0
```

I tried running `helm list --filter fluvio-sys --output json` to see what the output was, and it in fact was not JSON:

```
$ helm list --filter fluvio-sys --output json
Error: Kubernetes cluster unreachable: Get "https://172.17.0.3:8443/version?timeout=32s": net/http: TLS handshake timeout
```

After a little bit of searching, it seems like it may be related to https://github.com/rancher/k3s/issues/1126 but I'm not sure. I'm wondering whether the `kubectl` context was not initialized properly. Since encountering this error, I haven't been able to reproduce it, something happened and now `helm list` gives the expected output. I wanted to make a note of it in case we see it again in the future. I'm wondering, if it is a `kubectl` context issue, whether performing the `fluvio cluster set-minikube-context` step during the installation will help.

**Comment by sehz** [comment]
See https://github.com/infinyon/fluvio/pull/308.  

Can't seem to find original logs.

This was due to helm regression.  Please update your helm

**Comment by sehz** [comment]
This should be fixed with #308.
